src/main.cpp

```diff
@@ -37,7 +39,7 @@ int main(int argc, char **argv)
 
     SDL_ShowCursor(SDL_DISABLE);
     g_application.OnStart(argc, argv);
-    glClearColor(0.0f, 0.0f, 0.0f, 0.0f);
+    glClearColor(0.7f, 0.0f, 0.0f, 0.0f);
```

Comment:
Leave it black please ;)

Reply:
Actually I saw this, but it was too late :)